test/features/odi-certificates.feature

```diff
@@ -1,20 +1,19 @@
+@odi-certificates
 Feature: ODI Certificates
 
-    Scenario: Create a new test organisation for subsequent tests
+    Background:
```

Comment:
nice, it might be a good step to look into the rest api for doing the check, then you won't need to go through so many pages

https://data.qld.gov.au/api/3/action/organization_list
https://data.qld.gov.au/api/3/action/organization_show?id=aboriginal-and-torres-strait-islander-partnerships&include_users=false

Test Organisation would be id test-organisation